### ISSUE TITLE
Pull Request Guide - Instructions to rerun all validation don't work

### ISSUE THREAD

**Issue by felipepessoto** [dotnet/runtime#38997]
When I add a comment with "/azp run runtime" I received this error:
![image](https://user-images.githubusercontent.com/1336227/87052441-2a4c6300-c1d7-11ea-9a5b-d7070f4247c0.png)

I also can't find the "re-run all checks" button in the GitHub Checks tab

https://github.com/dotnet/runtime/blob/master/docs/pr-guide.md

**Comment by Gnbrkm41** [comment]
Closing then re-opening the PR also does the job.

**Comment by ViktorHofer** [comment]
@MattGal @danmosemsft what's the status on allowing external contributors to trigger /azp commands? IIRC we wanted to make that work?

**Comment by MattGal** [comment]
> @MattGal @danmosemsft what's the status on allowing external contributors to trigger /azp commands? IIRC we wanted to make that work?

I was actually unaware of this specific limitation but I guess it makes sense as it allows more than the PR trigger might run.  I'll ask around and reply back if I find useful stuff but I expect there'd always need to be some gate on who can issue such commands, otherwise writing a bot to just mess with it would be way too easy.

**Comment by felipepessoto** [comment]
When contributing to corefx I was able to trigger the pipeline using dotnet-bot commands: https://github.com/dotnet/corefx/pull/17715

**Comment by Gnbrkm41** [comment]
FWIW: https://github.com/dotnet/runtime/issues/718#issuecomment-564311630

**Comment by felipepessoto** [comment]
Makes sense. It may be the case of just changing the docs: https://github.com/dotnet/runtime/blob/master/docs/pr-guide.md
